package.json

```diff
@@ -2,14 +2,20 @@
   "name": "appium-plugins-monorepo",
   "private": true,
   "devDependencies": {
-    "@appium/eslint-config-appium": "4.7.4",
+    "@appium/eslint-config-appium": "5.0.0",
     "@babel/core": "7.16.0",
+    "@babel/eslint-parser": "7.16.3",
     "@babel/plugin-proposal-class-properties": "7.16.0",
-    "appium": "2.0.0-beta.19",
+    "@types/mocha": "9.0.0",
+    "@types/node": "16.11.9",
+    "appium": "2.0.0-beta.22",
     "chai": "4.3.4",
     "chai-as-promised": "7.1.1",
     "copyfiles": "2.4.1",
-    "eslint": "8.3.0",
+    "eslint": "7.32.0",
```

Comment:
unfortunately i had to downgrade this due to the appium-eslint-config@5.0.0 peer dep

Reply:
`eslint@8` is not going to be workable until either a) `eslint-plugin-promise` supports `eslint@8`, or we drop `eslint-plugin-promise`.

tbh not sure why many of the rules in `eslint-plugin-promise` aren't in eslint itself